docs-source/content/faq/external-clients.md

```diff
@@ -195,6 +234,23 @@ spec:
 |`spec.ports.nodePort`|The external port that clients will use. This must match the external port that's configured on the WebLogic configured channels/network access points. By default, Kubernetes requires that this value range from `30000` to `32767`.|
 |`spec.ports.port` and `spec.targetPort`|These must match the port that's configured on the WebLogic configured channel/network access points.|
 
+#### Enabling unknown host access
+
+##### When is it necessary to enable unknown host access?
+
+If a source WebLogic Server attempts to initiate an EJB, JMS, or JTA connection with a target WebLogic Server, then the target WebLogic Server will reject the connection by default if it cannot find the source server's listen address in its DNS. Such a failed connection attempt can yield log messages or exceptions like "...RJVM has already been shutdown..." or "...address was valid earlier, but now we get...". 
+
+This means that it's usually necessary to enable unknown host access on an external WebLogic Server so that it can support EJB, JMS, or JTA communication that is initated by an operator hosted WebLogic Server. For example, if an operator hosted WebLogic Server with service address 'mydomainuid-myservername' initiates a JMS connection to a remote WebLogic Server, then the remote server will implicitly attempt to lookup 'mydomainuid-myservername' in its DNS as part of the connection setup, and this lookup will typically fail.
+
+Similarly, this also means that it's necessary to enable unknown host access on an operator hosted WebLogic Server that accepts EJB or JMS connection requests from external WebLogic Servers when the external WebLogic Server's listen addresses cannot be resolved by the DNS running in the Kuberneters cluster.
+
+##### How to enable unknown host access
+
+To enable an 'unknown host' source WebLogic Server to initiate EJB, JMS, or JTA communication with a target WebLogic Server:
+  * Set the `weblogic.rjvm.allowUnknownHost` Java system property to `true` on each target WebLogic Server.
+    * For operator hosted WebLogic Servers, you can set this property by including `-Dweblogic.rjvm.allowUnknownHost=true` in the `JAVA_OPTIONS` [Domain environment variable]({{< relref "/userguide/managing-domains/domain-resource#jvm-memory-and-java-option-environment-variables" >}}) defined in the domain resource's `spec.serverPod.env` attribute.
+  * Also apply patch 30656708 on each target WebLogic Server for versions 12.2.1.4 (PS4) or earlier.
```

Comment:
`or earlier` -> `12.2.1.3 (PS3)` since we only have patch for 12.2.1.4 and 12.2.1.3.

This section LGTM. @lennyphan could you also review this? Thanks.

Reply:
@alai8 Can we leave it as 'or earlier'? Since the target WLS may be external to Kubernetes, it's very possible it'll be on an older version and the customer will therefore want a backport.